client/app/deploy/ServerRoleController.js

```diff
@@ -0,0 +1,22 @@
+/* TODO: enable linting and fix resulting errors */
```

Comment:
Outstanding todo

Reply:
Yeah, I think we need a plan for this. I am really not feeling these es-lint rules.

Reply:
> Line 84 exceeds the maximum line length of 150
> EOL at end of file
> No blank link at start of scope

Some of this stuff is silly.

Reply:
Not trailing spaces ... the list just goes on and on.